charts/lfx-platform/templates/authelia-client-secrets.yaml

```
@@ -0,0 +1,42 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+{{- if and .Values.authelia_enabled .Values.authelia_client_generation.enabled }}
+
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: authelia-clients
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "lfx-platform.labels" . | nindent 4 }}
+    app.kubernetes.io/component: authelia
+  annotations:
+    helm.sh/hook: pre-install
+    helm.sh/hook-weight: "0"
+    helm.sh/hook-delete-policy: before-hook-creation
+data:
+  {{- range $index, $client := .Values.authelia_client_generation.clients }}
+  {{ $client }}: {{ randAlphaNum 32| b64enc }}
```

Can you change this to use `randBytes` as it's already returns it's value as base-64 encoded, and allows us to specify the exact number of [recommended bytes](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#input-limits-of-bcrypt) for bcrypt?
```suggestion
  {{ $client }}: {{ randBytes 72 }}
```

---

I ended up rewriting this not to pull a base 64 encoded value (and instead generate a password for each user/client and then bcrypt | b64enc one and just b64enc the same value in another secret.